apps/docs/content/guides/database/prisma-troubleshooting.mdx

```diff
@@ -0,0 +1,247 @@
+---
+id: 'prisma-troubleshooting'
+title: 'Troubleshooting prisma errors'
+description: 'Prisma error troubleshooting'
+breadcrumb: 'ORM Quickstarts'
+---
+
+This guide addresses common Prisma errors that you might encounter while using Supabase.
+
+<Admonition type="note">
+
+A full list of errors can be found in [Prisma's official docs](https://www.prisma.io/docs/orm/reference/error-reference)
+
+</Admonition>
+
+## Understanding connection string parameters:
+
+Unlike other libraries, Prisma lets you configure [its settings](https://www.prisma.io/docs/orm/overview/databases/postgresql#arguments) through special options appended to your connection string. These options are called "query parameters."
+
+These options, called "query parameters," can be used to address specific errors.
+
+```md
+# Example of query parameters
+
+connection_string.../postgres?KEY1=VALUE&KEY2=VALUE&KEY3=VALUE
+```
+
+# Errors
+
+## ... prepared statement "" already exists
+
+Supavisor in transaction mode (port 6543) does not support [prepared statements](https://www.postgresql.org/docs/current/sql-prepare.html), which Prisma will try to create in the background.
+
+### Solution:
```

Comment:
Using the same heading multiple times will cause problems with ToC highlighting:

<img width="272" alt="image" src="https://github.com/user-attachments/assets/050b6221-1e03-49a4-9e31-cc7200f59eda">

You can fix this by adding a custom ID, like this:

```
### Solution: [#solution-prepared-statement-exists]
```

Reply:
@charislam, I think there's a bug. I added unique names and instead of highlighting the correct section, it would always bold the titles a few sections down.